README.md

```diff
@@ -17,21 +17,32 @@
 
 ### How do I set the Organization to Public?
 
-If you'd like to tell the world which organizations you belong to, you can display the avatars of the organizations on your profile. <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership">Click me</a> for example. 
+If you'd like to tell the world which organizations you belong to, you can display the avatars of the organizations on your profile. Check the <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership">GitHub Docs</a> for how to do this. OR you can also:
+
+- Click on the `View all` text.
+
+![image](https://user-images.githubusercontent.com/51391473/133660173-8b63685c-8e7f-48dc-8a7e-e97335941137.png)
+
+- Set it to the public option.
+
+![image](https://user-images.githubusercontent.com/51391473/133660356-65cd3131-ed6e-4d1e-9bcd-cf6412b76662.png)
+
+- That's it! Noww your profile in the organization is visible to anyone! 🎉
```

Comment:
Small typo, should be Now :) , LGTM but we should wait for @vinzvinci review before merging :)

Reply:
Great catch, thank you! It's fixed.